build/workbook.tpl.json

```diff
@@ -621,7 +621,8 @@
             "type": 3,
             "content": {
               "version": "KqlItem/1.0",
-              "query": ${kql_summary_workbook_reliability_score},
+              "query": ${kql_summary_workbook_reliability_score
+              },
```

Comment:
It's better to remove LF at end of line. But this is not mandatory.
```
"query": ${kql_summary_reliability_score_by_resource_environment <---- better remove CRLF
},
```

Reply:
done